tests/Integrations/Laravel/V5_7/EloquentTest.php

```diff
@@ -0,0 +1,144 @@
+use DDTrace\Tests\Common\TracerTestTrait;
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
+
+class EloquentTest extends WebFrameworkTestCase
+{
+    use TracerTestTrait, SpanAssertionTrait;
+
+    protected static function getAppIndexScript()
+    {
+        return __DIR__ . '/../../../Frameworks/Laravel/Version_5_7/public/index.php';
+    }
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->connection()->exec("DELETE from users where email LIKE 'test-user-%'");
+    }
+
+    public function testGet()
+    {
+        $traces = $this->tracesFromWebRequest(function () {
+            $spec  = GetSpec::create('Eloquent get', '/eloquent/get');
+            $this->call($spec);
+        });
+        $this->assertExpectedSpans($traces, [
+            SpanAssertion::exists('laravel.request'),
+            SpanAssertion::exists('PDO.__construct'),
+            SpanAssertion::exists('PDO.exec'),
+            SpanAssertion::exists('PDO.prepare'),
+            SpanAssertion::exists('PDOStatement.execute'),
+            SpanAssertion::exists('PDO.prepare'),
+            SpanAssertion::exists('PDOStatement.execute'),
+            SpanAssertion::exists('PDO.prepare'),
+            SpanAssertion::exists('PDOStatement.execute'),
+            SpanAssertion::build(
+                'eloquent.get',
+                '',
+                'sql',
+                'select * from `users`'
+            )->withExactTags([
+                'sql.query' => 'select * from `users`',
+            ]),
+        ], true);
+    }
+
+    public function testInsert()
+    {
+        $traces = $this->tracesFromWebRequest(function () {
+            $spec  = GetSpec::create('Eloquent insert', '/eloquent/insert');
+            $response = $this->call($spec);
+        });
+        $this->assertExpectedSpans($traces, [
+            SpanAssertion::exists('laravel.request'),
+            SpanAssertion::exists('PDO.__construct'),
+            SpanAssertion::exists('PDO.exec'),
+            SpanAssertion::exists('PDO.prepare'),
+            SpanAssertion::exists('PDOStatement.execute'),
+            SpanAssertion::exists('PDO.prepare'),
+            SpanAssertion::exists('PDOStatement.execute'),
+            SpanAssertion::exists('PDO.prepare'),
+            SpanAssertion::exists('PDOStatement.execute'),
+            SpanAssertion::build(
+                'eloquent.insert',
+                '',
+                'sql',
+                'insert ...'
+            )->withExactTags([
+                'sql.query' => 'insert ...',
+            ]),
+        ], true);
+    }
+
+    public function testUpdate()
+    {
+        $this->connection()->exec("insert into users (email) VALUES ('test-user-updated@email.com')");
+        $traces = $this->tracesFromWebRequest(function () {
+            $spec  = GetSpec::create('Eloquent update', '/eloquent/update');
+            $response = $this->call($spec);
```

Comment:
Does `$response` need to get used somehow?

Reply:
It was just useful to print it out, let me remove from everywhere though, good idea